### PR TITLE
Python add NativeTokenFeature

### LINKED ISSUE #1433: Python: Add Native Token Feature
https://github.com/iotaledger/iota-sdk/issues/1431 just for Python

### PR DESCRIPTION
**Pull request by Thoralf-M**
# Description of change

Added the native token feature and replaced the native_token field of outputs with it where needed.

## Links to any relevant issues

Closes https://github.com/iotaledger/iota-sdk/issues/1433

## Type of change

- Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: bindings/python/iota_sdk/types/output.py [hunk 1]
```diff
@@ -110,16 +104,15 @@ class AccountOutput:
             decoder=deserialize_unlock_conditions
         ))
     features: Optional[List[Union[SenderFeature,
-                            MetadataFeature]]] = field(default=None,
-                                                       metadata=config(
-                                                           decoder=deserialize_features
-                                                       ))
+                            MetadataFeature, NativeTokenFeature]]] = field(default=None,
```
Comment (thibault-martinez) [inline bindings/python/iota_sdk/types/output.py]: Should be in the Foundry instead of Account?

File: bindings/python/iota_sdk/types/feature.py [hunk 2]
```diff
@@ -18,15 +18,17 @@ class FeatureType(IntEnum):
...
+        Staking (6): The staking feature.
     """
     Sender = 0
     Issuer = 1
     Metadata = 2
     Tag = 3
-    BlockIssuer = 4
-    Staking = 5
+    NativeToken = 5
+    BlockIssuer = 5
```
Comment (thibault-martinez) [inline bindings/python/iota_sdk/types/feature.py:30]: Two times 5

File: bindings/python/iota_sdk/types/output.py (hunk 1 above)
Comment (Thoralf-M) [reply bindings/python/iota_sdk/types/output.py]: Yes good catch

**Review by thibault-martinez — COMMENTED** [review]
Do you want to update `consolidate_outputs.py` and `05_get_address_balance.py`?

File: bindings/python/examples/client/05_get_address_balance.py [hunk 3]
```diff
@@ -35,8 +35,10 @@
 for output_with_metadata in outputs:
     output = output_with_metadata.output
     total_amount += output.amount
-    if output.native_tokens:
-        native_tokens.append(output.native_tokens)
+    native_token = [feature for feature in output.features if feature['type']
```
Comment (thibault-martinez) [inline bindings/python/examples/client/05_get_address_balance.py]: Would `feature.type` work instead?

Comment (Thoralf-M) [reply bindings/python/examples/client/05_get_address_balance.py]: Maybe, can't test at the moment since all is broken

**Review by thibault-martinez — APPROVED** [review]

**Review by Alex6323 — APPROVED** [review]

**Review by kwek20 — APPROVED** [review]

**Pull request merged**